traffic_ops/client/region.go

```diff
@@ -159,3 +160,28 @@ func (to *Session) GetRegionByNamePath(name string) ([]tc.RegionName, ReqInf, er
 	}
 	return resp.Response, reqInf, nil
 }
+
+// DELETE a Region
+func (to *Session) DeleteRegionByName(id *int, name *string) (tc.Alerts, ReqInf, error) {
```

Comment:
godoc is off from name of func and nit but I wonder if this should be called DeleteRegion as you can delete it by id or name

Reply:
Fixed GoDoc and renamed in 2ada94e8f2.